src/main.c

```diff
@@ -66,6 +66,12 @@ USAGE \
 "                                <digest> is the X509 certificate's sha256 sum.\n" \
 "                                This option can be used multiple times to trust\n" \
 "                                several certificates.\n" \
+"  --insecure-ssl                Do not disable insecure SSL protocols/ciphers.\n" \
+"                                If your server requires a specific cipher, consider\n" \
+"                                using --cipher-list instead.\n" \
+"  --cipher-list=<list>          openssl ciphers to use. If default does not work\n" \
+"                                you can try openssl s_client to test alternative\n" \
+"                                ciphers.\n" \
```

Comment:
This text is good and understandable! But could you show the expect list format for newbies? (With an example for instance `"HIGH:!MD5:!RC4"`).

Also could you precise that it works only when `--insecure-ssl` is not set?